src/main.rs

```diff
@@ -1,9 +1,28 @@
 use swayipc::reply::{Event, NodeLayout, NodeType, WindowChange};
 use swayipc::{Connection, EventType};
 
-fn switch_splitting(conn: &mut Connection) -> Result<(), String> {
+use clap::{app_from_crate, crate_authors, crate_description, crate_name, crate_version, Arg};
+
+fn switch_splitting(conn: &mut Connection, workspaces: &[String]) -> Result<(), String> {
+    // Check if focused workspace is in "allowed list".
+    // If `workspaces` is empty, skip allow all workspaces.
+    if !workspaces.is_empty() {
+        for workspace in conn
+            .get_workspaces()
+            .map_err(|_| "get_workspaces() failed")?
+        {
+            if workspace.focused {
+                if workspaces.contains(&workspace.name) {
```

Comment:
Should we check .num instead?

Reference: https://github.com/nwg-piotr/autotiling/pull/22#pullrequestreview-563904491

Reply:
It turns out I've been using this feature the wrong way. I'll open new PR.